core/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -18,9 +18,9 @@
 package org.apache.spark.internal
 
 import org.apache.log4j.{Level, LogManager, PropertyConfigurator}
+import org.apache.spark.deploy.{SparkShellType, SparkSubmit}
 import org.slf4j.{Logger, LoggerFactory}
 import org.slf4j.impl.StaticLoggerBinder
-
 import org.apache.spark.util.Utils
```

Comment:
@felixcheung It seems that you saw my old commit for some reason. I think I changed the import order in the latest commit. Thanks!